docs/section16/Rule16-4.md

```diff
@@ -0,0 +1,34 @@
+# Elevators – Rule 16-4  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-4  
+**Rule Description:** The elevator cab lights shall be modeled with the same schedule as the elevator motor.  
+**Rule Assertion:** P-RMD = Expected Value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16  
+**Data Lookup:** None  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+N/A
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- For each elevator in the proposed RMD: `for elevator in P_RMD...elevators`
+  - Get the modeled schedule for the elevator motor use in the proposed: `motor_use_schedule_p = elevator_p.cab_motor_multiplier_schedule`
+  - Get the modeled schedule for the elevator cab lighting use in the proposed: `cab_lighting_schedule_p = elevator_p.cab_lighting_multiplier_schedule`
+  - Compare the schedules: `comparison_data = compare_schedules(cab_lighting_schedule_p, motor_use_schedule_p):`  
```

Comment:
Missing the third input to the compare_schedules function. Discussed with Jackson; he will discuss with Weili whether this can be made an optional input. If can't be optional then this will be an 8760 schedule of all 1's